### PR TITLE
[Playwright] LPD-24934 Functional test for LPD-16313

### PR DESCRIPTION
**Pull request by gergelyszalay**
Hi,

May I ask to review my PR?

The link for [figma](https://www.figma.com/file/YMcgShbEYvN2XMPPpc154t/LPD-16311-Define-and-manage-Site-default-D%26M-permissions?type=design&node-id=47-7787&mode=design)

Thank you!

### CODE REVIEW

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:test:stop

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:test:stop

**Comment by gergelyszalay** [comment]
ci:test:rrelevant

**Comment by jkappler** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by gergelyszalay** [comment]
ci:report:2568445688

**Comment by gergelyszalay** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:test:playwright

**Comment by boton** [comment]
It is failing in the CI https://github.com/liferay-content-management/liferay-portal/pull/5440#issuecomment-2117305670
image

@gergelyszalay is it working on your local?

**Review by boton — CHANGES_REQUESTED** [review]
It is failing in my local
image and in [the CI](https://github.com/liferay-content-management/liferay-portal/pull/5440#issuecomment-2117305670).

I prefer don't review the code until it the tests are passing

**Comment by gergelyszalay** [comment]
Hi @boton , it was working, but i will check it again. thanks for looking into

**Comment by gergelyszalay** [comment]
ci:test:playwright

**Comment by gergelyszalay** [comment]
force-pushed my tests, it is running again on the local, and the fileEntry disappeared from the ci:playwright report. Thanks!

**Review by boton — CHANGES_REQUESTED** [review]
There are failing in my local 
image

The problem looks like the test tried to click on the actions button but the JS isn't ready yet 
image

**Comment by jkappler** [comment]
ci:test:playwright

**Comment by jkappler** [comment]
@boton, don't see failures related to the changes

File: modules/test/playwright/pages/document-library-web/DocumentLibraryPage.ts [hunk 1]
```diff
@@ -27,14 +28,59 @@ export class DocumentLibraryPage {
...
+	async changeTab(tabName: string) {
+		await this.page.getByRole('link', {name: tabName}).click();
+	}
+
+	async changeView(viewName: string) {
+		await this.page
+			.getByLabel('Select View, Currently Selected: ')
+			.waitFor();
+		await this.page.getByLabel('Select View, Currently Selected: ').click();
+		await this.page.getByRole('menuitem', {name: viewName}).click();
```
Comment (boton) [inline modules/test/playwright/pages/document-library-web/DocumentLibraryPage.ts:49]: Use the `clickAndExpectToBeVisible` around here we are having the same problem, the `this.page.getByRole('menuitem', {name: viewName})` isn't visible when we launch the click

**Review by boton — CHANGES_REQUESTED** [review]
I'm so sorry @gergelyszalay but still failing on my local.

image

I think it passed the CI but was still flaky.

I suggest you use something like

```sh
npm run test -- -g "LPD-16313" --reporter list --repeat-each 10
```

to check the test multiple times.

**Comment by gergelyszalay** [comment]
ci:test:playwright

**Comment by boton** [comment]
It's almost done I'm investigating how to remove `timeouts` to reduce the flakiness

image

**Pull request closed without merging**